ethereum/executionlayer/src/test/java/tech/pegasys/teku/ethereum/executionlayer/FuluExecutionClientHandlerTest.java

```diff
@@ -0,0 +1,209 @@
+            dataStructureUtil.randomBytes32(),
+            dataStructureUtil.randomEth1Address(),
+            Optional.empty(),
+            Optional.of(List.of()),
+            dataStructureUtil.randomBytes32());
+    final Optional<PayloadAttributesV3> payloadAttributes =
+        PayloadAttributesV3.fromInternalPayloadBuildingAttributesV3(Optional.of(attributes));
+    final ForkChoiceUpdatedResult responseData =
+        new ForkChoiceUpdatedResult(
+            new PayloadStatusV1(
+                ExecutionPayloadStatus.ACCEPTED, dataStructureUtil.randomBytes32(), ""),
+            dataStructureUtil.randomBytes8());
+    final SafeFuture<Response<ForkChoiceUpdatedResult>> dummyResponse =
+        SafeFuture.completedFuture(Response.fromPayloadReceivedAsJson(responseData));
+    when(executionEngineClient.forkChoiceUpdatedV3(forkChoiceStateV1, payloadAttributes))
+        .thenReturn(dummyResponse);
+    final SafeFuture<tech.pegasys.teku.spec.executionlayer.ForkChoiceUpdatedResult> future =
+        handler.engineForkChoiceUpdated(forkChoiceState, Optional.of(attributes));
+    verify(executionEngineClient).forkChoiceUpdatedV3(forkChoiceStateV1, payloadAttributes);
+    assertThat(future).isCompletedWithValue(responseData.asInternalExecutionPayload());
+  }
+
+  @Test
+  void engineGetBlobs_shouldCallGetBlobsV2() {
+    final ExecutionClientHandler handler = getHandler();
+    final int maxBlobsPerBlock =
+        SpecConfigDeneb.required(spec.getGenesisSpecConfig()).getMaxBlobsPerBlock();
+    final List<VersionedHash> versionedHashes =
+        dataStructureUtil.randomVersionedHashes(maxBlobsPerBlock);
+    final BlobsCellBundle blobsCellBundle =
+        dataStructureUtil.randomBlobsCellBundle(maxBlobsPerBlock);
+    final UInt64 slot = dataStructureUtil.randomUInt64(1_000_000);
+    final List<BlobAndProofV2> responseData =
+        IntStream.range(0, blobsCellBundle.getBlobs().size())
+            .mapToObj(
+                i ->
+                    new BlobAndProofV2(
+                        blobsCellBundle.getBlobs().get(i).getBytes(),
+                        blobsCellBundle
+                            .getProofs()
+                            .subList(i * CELLS_PER_EXT_BLOB, (i + 1) * CELLS_PER_EXT_BLOB)
+                            .stream()
+                            .map(KZGProof::getBytesCompressed)
+                            .toList()))
+            .toList();
+
+    final SafeFuture<Response<List<BlobAndProofV2>>> dummyResponse =
+        SafeFuture.completedFuture(Response.fromPayloadReceivedAsJson(responseData));
+    when(executionEngineClient.getBlobsV2(versionedHashes)).thenReturn(dummyResponse);
+    final SafeFuture<List<BlobAndCellProofs>> future =
+        handler.engineGetBlobsV2(versionedHashes, slot);
+    verify(executionEngineClient).getBlobsV2(versionedHashes);
+    final BlobSchema blobSchema =
+        spec.atSlot(slot).getSchemaDefinitions().toVersionFulu().orElseThrow().getBlobSchema();
```

Comment:
`SchemaDefinitionsFulu.required` please. Better to put in `schemaDefinitionsFulu` as a global variable in class